### DISCUSSION TITLE
group_concat() multi columns OOM

### DISCUSSION THREAD

**Discussion by 16601746291** [apache/doris#19218] [RFC]
when i execute sql  that model like :
 select column01, group_concat(column02, ','  order by update_time desc) as new columnA
           ,new columnB
           ,new columnC
           ,new columnD
           ,......
           ,new columnZ
 from table 
 group by column01

total 22 columns ,like this:
![image](https://user-images.githubusercontent.com/49640575/235189250-37efe02e-249b-4a14-a0a2-6204c0fd1504.png)

result OOM，How to optimize script statement?

**Comment by yagagagaga** [comment]
I need more information to help you:
1. What is your Doris version?
2. Can you provide the profile of this case? You can enable profile by executing this sql `SET enable_profile=true;`. For more information about profile, you can see [SHOW-QUERY-PROFILE](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Show-Statements/SHOW-QUERY-PROFILE/)
3. Can you provide the table DDL?
